src/Piling/PilingParameter.cs

```diff
@@ -24,6 +24,14 @@ private PilingParameter(ISharedParameterManager manager)
             _manager = manager ?? throw new ArgumentNullException(nameof(manager));
         }
 
+        public void SetParameterValue(Parameter parameter, double value)
```

Comment:
Not clear this will only set the parameter if the name matches - this is hidden inside the implementation. Needs either documenting or renaming

Reply:
Valid - suggesting return bool to indicate if parameter was updated, and also xml documentation to interface <inheritdoc /> tag on concert implementation.